src/database/models/accommodationlikes.js

```diff
@@ -0,0 +1,12 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const accommodationlikes = sequelize.define('accommodationlikes', {
```

Comment:
@VedaDom Does this mean that there's also an accommodationDislikes table? I suggest finding another generic name. It would be better to have two columns i.e. like & dislike since it will be easier to sum total likes and dislikes.